scripts/getSecrets.js

```diff
@@ -27,6 +27,12 @@ const getSecrets = async (name, region) => {
     if (!data.SecretString) {
       throw new Error('Secrets not found')
     }
+    else if(data.SecretString === undefined){
```

Comment:
These are not needed as the `!data.SecretString` condition is already taking care of them.